### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by ygrignon-sfdc**
[Related and Nested Entities] Fixed URN for entity functions and added tests
[JSON] Fixed/updated AttributeValue handling based on JSON Profile Version 1.1 and added tests
[AttributeSelector] content namespace for xpath evaluation

### CODE REVIEW

**Comment by ygrignon-sfdc** [comment]
This PR fixes Issue #6

**Comment by ygrignon-sfdc** [comment]
@pdragosh @bcruz32 can you guys review this PR? Thanks!

**Comment by pdragosh** [comment]
Ok thanks for the PR - I'd like to look more closely at this. ETA Friday afternoon.

File: xacml-pdp/src/test/java/com/att/research/xacmlatt/pdp/test/entity/EntityTest.java [hunk 1]
```diff
@@ -12,12 +12,20 @@
 import com.att.research.xacml.api.pdp.PDPEngine;
 import com.att.research.xacml.api.pdp.PDPEngineFactory;
 import com.att.research.xacml.std.annotations.*;
+import com.att.research.xacml.std.dom.DOMUtil;
 import com.att.research.xacmlatt.pdp.policy.expressions.ForAny;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
```
Comment (dcruz32) [inline xacml-pdp/src/test/java/com/att/research/xacmlatt/pdp/test/entity/EntityTest.java:18]: I don't see a use for this import

File: xacml-pdp/src/test/java/com/att/research/xacmlatt/pdp/test/entity/EntityFunctionsTest.java [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+import com.att.research.xacml.std.datatypes.DataTypes;
+import com.att.research.xacml.std.datatypes.XPathExpressionWrapper;
+import com.att.research.xacml.std.dom.DOMStructureException;
+import com.att.research.xacml.std.dom.DOMUtil;
+import com.att.research.xacmlatt.pdp.eval.EvaluationContext;
+import com.att.research.xacmlatt.pdp.policy.*;
+import com.att.research.xacmlatt.pdp.std.StdEvaluationContext;
+import com.att.research.xacmlatt.pdp.std.StdFunctionDefinitionFactory;
+import com.att.research.xacmlatt.pdp.std.functions.*;
+import org.junit.Ignore;
```
Comment (dcruz32) [inline xacml-pdp/src/test/java/com/att/research/xacmlatt/pdp/test/entity/EntityFunctionsTest.java:28]: Remove unused import

**Review by dcruz32 — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -41,6 +41,12 @@
             <organization>ATT</organization>
             <organizationUrl>http://www.att.com</organizationUrl>
         </developer>
+        <developer>
```
Comment (pdragosh) [inline pom.xml:44]: Thanks for your contributions, but I cannot add you here as you don't work for AT&T.

File: xacml/src/main/java/com/att/research/xacml/std/StdAttributeValue.java [hunk 4]
```diff
@@ -9,6 +9,8 @@
 import com.att.research.xacml.api.Identifier;
 import com.att.research.xacml.util.ObjUtil;
 
+import java.util.Collection;
```
Comment (pdragosh) [inline xacml/src/main/java/com/att/research/xacml/std/StdAttributeValue.java:12]: Is this an unused import? Not sure why its added.

**Comment by ygrignon-sfdc** [comment]
@pdragosh can we get this over the line? Thanks!

**Comment by pdragosh** [comment]
Working on it, we had some malware pushing bad commits over the last couple of weeks. I was also out on vacation. I just wanted to look at it locally real quick.

**Comment by ygrignon-sfdc** [comment]
Sorry to be pushy @pdragosh but we are waiting on the release of the fix to issue #6 included in this PR.

**Comment by pdragosh** [comment]
You're not being pushy, I meant to finish this out this morning. Got stuck down a rabbit-hole working on another problem.

**Comment by pdragosh** [comment]
I think I screwed up though, I missed out on merging it.

**Pull request merged**